client/state/inline-help/selectors/get-inline-help-search-results-for-query.js

```diff
@@ -22,3 +22,12 @@ export default function getInlineHelpSearchResultsForQuery( state ) {
 	const allResults = get( state, 'inlineHelp.searchResults.search.items' );
 	return get( allResults, [ searchQuery ], [] );
 }
+
+export function getInlineHelpAdminSectionSearchResultsForQuery( state ) {
```

Comment:
All the inline-help selectors are located in their individual files, so let's move this to its individual home as well :)

Reply:
I wonder how we ended up with such a pattern ? Will change it for conformity even though I don't completely agree with this approach.